### PR TITLE
fix: update doc to include pipelineId in updateCommitStatus

### PR DESCRIPTION
**Pull request by d2lam**
Related: https://github.com/screwdriver-cd/screwdriver/issues/471

### CODE REVIEW

**Comment by d2lam** [comment]
The build is failing because it's pulling in `screwdriver-data-schema@16.8.5` despite package.json specifies `^16.9.2`. Super weird!

**Comment by tkyi** [comment]
I don't get why it does that :[

**Comment by ian-fox** [comment]
That's really weird!

**Comment by d2lam** [comment]
Rolling back to `node:6` because of https://github.com/npm/npm/issues/16866
`package-lock.json` is not updated, so it keeps pulling `16.8.5` from the lock file.

File: package.json [hunk 1]
```diff
@@ -44,6 +44,6 @@
   },
   "dependencies": {
     "joi": "^10.0.5",
-    "screwdriver-data-schema": "^16.0.2"
+    "screwdriver-data-schema": "16.9.2"
```
Comment (minz1027) [inline package.json]: You don't need to pin now.

**Pull request merged**